src/Camera.cc

```diff
@@ -1,22 +1,29 @@
 #include "Camera.h"
+#include "QSettingsCached.h"
```

Comment:
Unfortunately that's not possible, `Camera.cc` is part of the core which may not depend on Qt, so we have to find a way to pass that setting along from outside.

There's currently only some hints in `CMakeLists.txt` so it's not very obvious, hopefully this can be made more clear at some point by getting the source files better organized. 
https://github.com/openscad/openscad/blob/edd69c2792b191bbd577fcc0acf2bbd836aa0f76/CMakeLists.txt#L451-L453

Reply:
OK. Will leave Camera.cc alone.

Reply:
How about the following:
- remove dubois.h and dubois.cc files
- add function duboisColor(), float eyeSeparation and bool anaglyph to class Renderer
Opinion?